core/src/main/java/io/aiven/kafka/tieredstorage/manifest/index/AbstractChunkIndex.java

```diff
@@ -108,6 +109,17 @@ public Chunk findChunkForOriginalOffset(final int offset) {
         );
     }
 
+    @Override
+    public List<Chunk> chunksForRange(final BytesRange bytesRange) {
+        Chunk current;
+        final var result = new ArrayList<Chunk>();
+        for (int i = bytesRange.from; i < bytesRange.to && i < originalFileSize; i += current.originalSize) {
```

Comment:
Let's make it explicit: do we calculate the prefetch range on the original or transformed size?